portfolio/pom.xml

```diff
@@ -45,7 +45,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>chrismohri-step-2020</deploy.projectId>
+          <deploy.projectId>chrismohri-step-2020-2</deploy.projectId>
```

Comment:
tiny nit: this still has the TODO: that you can remove

Reply:
removed